### PR TITLE
feat:PostgreSQLAutogeneratedPassword

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description

Used the auto-generated PosgtreSQL credentials.

### CODE REVIEW

File: charts/bpndiscovery/templates/secret.yaml [hunk 1]
```diff
@@ -19,16 +19,33 @@
...
+#  postgres-password: {{ $secretPassword | quote }}
+#  password: {{ $secretPassword | quote }}
   SPRING_DATASOURCE_URL: {{ printf "jdbc:postgresql://%s-postgresql:%v/%s" .Release.Name .Values.postgresql.service.ports.postgresql .Values.postgresql.auth.database | b64enc }}
   SPRING_DATASOURCE_USERNAME: {{ .Values.postgresql.auth.username | b64enc }}
-  SPRING_DATASOURCE_PASSWORD: {{ .Values.postgresql.auth.password | b64enc }}
+  {{ if $secret -}}
+  data:
+    postgres-password: {{ ( .Values.postgresql.auth.password | b64enc )  | default ( index $secret.data "postgres-password" ) | quote }}
+    password: {{ ( .Values.postgresql.auth.password | b64enc )  | default ( index $secret.data "postgres-password" ) | quote }}
+    SPRING_DATASOURCE_PASSWORD: {{ $secretPassword | quote }}
```
Comment (evegufy) [inline charts/bpndiscovery/templates/secret.yaml:41]: the secret values needs to be the values from [postgres-password](https://github.com/eclipse-tractusx/sldt-bpn-discovery/pull/71/files#diff-89698cfe1ba306a56daf2d3119401a86d8b361745a2cb49d7ee0ffd7365d442aR39), not just one 'randAlphaNum 32' like [$secretPassword](https://github.com/eclipse-tractusx/sldt-bpn-discovery/pull/71/files#diff-89698cfe1ba306a56daf2d3119401a86d8b361745a2cb49d7ee0ffd7365d442aR22) is currently defined

File: charts/bpndiscovery/templates/secret.yaml [hunk 2]
```diff
@@ -19,16 +19,33 @@
...
   {{- if .Values.enablePostgres }}
+#  postgres-password: {{ $secretPassword | quote }}
+#  password: {{ $secretPassword | quote }}
   SPRING_DATASOURCE_URL: {{ printf "jdbc:postgresql://%s-postgresql:%v/%s" .Release.Name .Values.postgresql.service.ports.postgresql .Values.postgresql.auth.database | b64enc }}
   SPRING_DATASOURCE_USERNAME: {{ .Values.postgresql.auth.username | b64enc }}
-  SPRING_DATASOURCE_PASSWORD: {{ .Values.postgresql.auth.password | b64enc }}
+  {{ if $secret -}}
+  data:
+    postgres-password: {{ ( .Values.postgresql.auth.password | b64enc )  | default ( index $secret.data "postgres-password" ) | quote }}
+    password: {{ ( .Values.postgresql.auth.password | b64enc )  | default ( index $secret.data "postgres-password" ) | quote }}
```
Comment (evegufy) [inline charts/bpndiscovery/templates/secret.yaml:40]: probably copy/paste error: here it's should be 'default $secret.data.password' not 'default ( index $secret.data "postgres-password"'

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by carslen** [comment]
Re-requested review from @evegufy by accident, sorry.

**Comment by tunacicek** [comment]
will be done in other PR https://github.com/eclipse-tractusx/sldt-bpn-discovery/pull/74

**Pull request closed without merging**